### ISSUE TITLE
Add validation set

### ISSUE THREAD

**Issue by Crista23** [Kyubyong/transformer#28]
I can see the model is training for a fixed number of epochs and there is no validation set. How to add one? Thank you!

**Comment by 4pal** [comment]
use tensorboard to monitor model accuracy and loss.use ctrl+c to stop the model --you can fire back train.py to resume training.run eval.py to see how you model has performed. this file is usually linked to validation sets which you can update as you wish.
